# Pick FC and FFLAGS from environmental variables
# March 8th 2016 B. Hajgato (Free Uviveristy Brussels - VUB)
--- qrupdate-1.1.2/Makeconf.old	2010-01-19 12:35:49.000000000 +0100
+++ qrupdate-1.1.2/Makeconf	2016-03-08 20:51:11.662603099 +0100
@@ -1,9 +1,9 @@
 # set this to your compiler's executable name (e.g. gfortran, g77)
-FC=gfortran
+FC?=gfortran
 # requested flags
-FFLAGS=-fimplicit-none -O3 -funroll-loops 
+FFLAGS?=-fimplicit-none -O3 -funroll-loops 
 # set if you need shared library
-FPICFLAGS=-fPIC
+FPICFLAGS=
 
 # BLAS library (only required for tests)
 BLAS=-lblas
